Point_set_processing_3/include/CGAL/internal/Voronoi_covariance_3/voronoi_covariance_3.h

```diff
@@ -148,6 +148,8 @@ namespace CGAL {
                     for(typename std::list<Vertex_handle>::iterator it = vertices.begin();
                         it != vertices.end(); ++it)
                     {
+                      if (dt.is_infinite(*it))
+                        continue;
                         Vector p = ((*it)->point() - v->point())/2;
```

Comment:
The indentation of the patch seems incorrect. It seems that this files uses four spaces by indentation level. The patch should follow the choice of the existing code.